### PR TITLE
Update master branch for production

### PR DESCRIPTION
**Pull request by SashaWeinstein**
**2+ reviewers** 🥧 
We want pluto version `22v2` on the deployed app, so it's time to merge dev into master so we can run the deployment workflow

>Note on emoji: I used a dessert to capture a sense of celebration and finality. The extended metaphor here is that the development is like a meal, and dessert is the nice part at the end

### CODE REVIEW

File: src/home.py [hunk 1]
```diff
@@ -68,4 +68,4 @@ def home():
     )
 
     st.header("Read more on Medium")
-    get_blogs()
+    # get_blogs()
```
Comment (mbh329) [inline src/home.py:71]: Remove comment

**Review by mbh329 — COMMENTED** [review]
What's the process for continued development on the QAQC app? After this is merged in, should we continue to branch off dev and merge into dev and periodically merge into master or  is the idea to use branch and merge off master from this point on?

File: src/cpdb/cpdb.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                x=df_bar_diff[VIZKEY[subcategory][view_type]["values"][map_option]],
+                y=df_bar_diff.index,
+                orientation="h",
+            ),
+            go.Bar(
+                name="Latest Version",
+                x=df[VIZKEY[subcategory][view_type]["values"][map_option]],
+                y=df.index,
+                orientation="h",
+                visible='legendonly'
```
Comment (mbh329) [inline src/cpdb/cpdb.py:126]: Single quotes still present

File: src/cpdb/cpdb.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+    fig3.update_layout(
+        width=1000,
+        height=1000,
+        title_text=f"Percentage Mapped of {view_type_title} by Version and {agency_type_title}"
+    )
+
+    fig3.update_xaxes(
+        title=f"Percentage",
+        tickformat= '.2%'
```
Comment (mbh329) [inline src/cpdb/cpdb.py:194]: single quote

File: src/home.py (hunk 1 above)
Comment (SashaWeinstein) [reply src/home.py:71]: I think the `get_blogs()` is broken, should be commented out

**Review by abrieff — APPROVED** [review]
This lgtm to me locally

**Review by mbh329 — APPROVED** [review]

**Review by Oysters1874 — COMMENTED** [review]
looks good to me

**Pull request merged**